model-optimizer/mo/front/kaldi/extractors/slice_ext.py

```diff
@@ -1,42 +0,0 @@
-"""
```

Comment:
Why did you remove this extractor? Are you sure that this operation is not used in other models?

Reply:
Before removing it I double checked. It's not used in other places. The only case when Slice nodes were created was for <ParallelComponent> which is now replaced by Split. Kaldi models itself don't have operation Slice. The old Slice was copypasted from Caffe.